package.json

```diff
@@ -94,6 +93,7 @@
     "enzyme": "^2.8.2",
     "eslint": "^3.15.0",
     "eslint-config-airbnb": "^13.0.0",
+    "eslint-import-resolver-webpack": "^0.8.3",
```

Comment:
Is this supposed to be in the sub-folder?

Reply:
As the sandbox just extends main `styled-components` config, I left this in the main `package.json` on purpose. Otherwise when you `npm install` the sandbox there are a lot of warnings about missing `peerDeps`. Should I move this to the sandbox anyway?

Reply:
ah, yea, that makes sense :+1: keep it then :smile: